libs/net/dal/Services/AVOverviewInstanceService.cs

```diff
@@ -93,15 +93,18 @@ public override AVOverviewInstance Update(AVOverviewInstance entity)
                 {
                     this.Context.Entry(s).State = EntityState.Deleted;
                 });
-                this.Context.Update(section);
+                this.Context.Entry(originalSections.First(x => x.Id == section.Id)).State = EntityState.Detached;
```

Comment:
Do we need this anymore?  We don't need this in other update functions.

Reply:
Updated and it seems we could remove it by using AsNoTracking.